### PR TITLE
feat: add signozkafkaexporter

### PR DESCRIPTION
**Pull request by dhawal1248**

### CODE REVIEW

**Review by prashant-shahi — APPROVED** [review]

**Comment by prashant-shahi** [comment]
Functionally looks good.
Except for new requirement of setting the environment variable `GODEBUG` to `netdns=go` for `cluster.local` endpoint(s) of kafka running in same cluster.

@dhawal1248 Also, can you resolve the test-pipeline issue?
Also, please do get code reviewed by someone from @SigNoz/backend team.

cc @ankitnayan

File: receiver/signozkafkareceiver/testdata/config.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+kafka:
```
Comment (prashant-shahi) [inline receiver/signozkafkareceiver/testdata/config.yaml:1]: use `signozkafkareceiver` instead

File: receiver/signozkafkareceiver/testdata/config.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  auth:
+    tls:
+      ca_file: ca.pem
+      cert_file: cert.pem
+      key_file: key.pem
+  metadata:
+    retry:
+      max: 10
+      backoff: 5s
+kafka/logs:
```
Comment (prashant-shahi) [inline receiver/signozkafkareceiver/testdata/config.yaml:17]: use `signozkafkareceiver` instead

File: exporter/signozkafkaexporter/testdata/config.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+kafka:
```
Comment (prashant-shahi) [inline exporter/signozkafkaexporter/testdata/config.yaml:1]: use `signozkafkaexporter` instead

File: receiver/signozkafkareceiver/testdata/config.yaml (hunk 1 above)
Comment (prashant-shahi) [reply receiver/signozkafkareceiver/testdata/config.yaml:1]:
```suggestion
signozkafkareceiver:
```

File: exporter/signozkafkaexporter/README.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+This exporter has been taken from opentelemetry-collector-contrib [https://github.com/open-telemetry/opentelemetry-collector-contrib]
+
+The exporter was imported at v0.79.0 [https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.79.0]
```
Comment (prashant-shahi) [inline exporter/signozkafkaexporter/README.md:3]: can you update this to v0.88.0?

File: receiver/signozkafkareceiver/README.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
+This exporter has been taken from opentelemetry-collector-contrib [https://github.com/open-telemetry/opentelemetry-collector-contrib]
+
+The receiver was imported at v0.79.0 [https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.79.0]
```
Comment (prashant-shahi) [inline receiver/signozkafkareceiver/README.md:3]: update this to v0.88.0

**Review by makeavish — APPROVED** [review]

**Comment by makeavish** [comment]
Approved only this commit: https://github.com/SigNoz/signoz-otel-collector/pull/195/commits/fe3dfe28f3ec52edbc9dff91b688aee67c2adc07

**Pull request merged**